### PR TITLE
fix: use alchemy to get gas price

### PR DESCRIPTION
**Pull request by peterMangrove**

### CODE REVIEW

File: packages/bot-updategas/config/test.json [hunk 1]
```diff
@@ -2,8 +2,7 @@
   "logLevel": "debug",
   "acceptableGasGapToOracle": 0.5,
   "constantOracleGasPrice": 6,
-  "runEveryXHours": 0.008,
-  "maxUpdateConstraint": {
-    "constant": 10
-  }
+  "network": "MATIC_MUMBAI",
+  "maxUpdateConstraint": {},
```
Comment (lnist) [inline packages/bot-updategas/config/test.json:6]: is the change of config of maxUpdateConstraint and runEveryXHours intentional? Then fine :)

File: packages/bot-updategas/config/default.json [hunk 2]
```diff
@@ -1,9 +1,7 @@
 {
   "logLevel": "debug",
   "acceptableGasGapToOracle": 0.5,
-  "oracleURL": "https://gasstation-mainnet.matic.network/v2",
-  "oracleURL_Key": "standard",
-  "oracleURL_subKey": "maxFee",
+  "network": "MATIC_MUMBAI",
```
Comment (lnist) [inline packages/bot-updategas/config/default.json:4]: do we know how reliable alchemy's oracle is compared to the gas station? I am fine with the change if we think alchemy is at least as good. This change removes the option to use the old oracle - in theory we could have kept both options - but no reason to do that if alchemy is good enough.

**Review by lnist — APPROVED** [review]

File: packages/bot-updategas/config/test.json (hunk 1 above)
Comment (peterMangrove) [reply packages/bot-updategas/config/test.json]: Copy paste mistake, reverted it.

File: packages/bot-updategas/config/default.json (hunk 2 above)
Comment (peterMangrove) [reply packages/bot-updategas/config/default.json:4]: old oracle to specific. Alchemy has more oracles.

**Pull request merged**